ui/src/dashboards/components/CellEditorOverlay.js

```diff
@@ -87,31 +124,44 @@ class CellEditorOverlay extends Component {
   }
 
   handleSaveCell() {
-    const {queriesWorkingDraft, cellWorkingType, cellWorkingName} = this.state
+    const {
+      queriesWorkingDraft,
+      cellWorkingType: type,
+      cellWorkingName: name,
+      axes,
+    } = this.state
+
     const {cell} = this.props
 
-    const newCell = _.cloneDeep(cell)
-    newCell.name = cellWorkingName
-    newCell.type = cellWorkingType
-    newCell.queries = queriesWorkingDraft.map(q => {
+    const queries = queriesWorkingDraft.map(q => {
       const timeRange = q.range || {upper: null, lower: ':dashboardTime:'}
       const query = q.rawText || buildInfluxQLQuery(timeRange, q)
-      const label = q.rawText ? '' : `${q.measurement}.${q.fields[0].field}`
 
       return {
         queryConfig: q,
         query,
-        label,
       }
     })
 
-    this.props.onSave(newCell)
+    this.props.onSave({
+      ...cell,
+      name,
+      type,
+      queries,
+      axes,
+    })
   }
 
   handleSelectGraphType(graphType) {
     this.setState({cellWorkingType: graphType})
   }
 
+  handleSelectDisplayOptions(isDisplayOptionsTabOpen) {
```

Comment:
i would find this clearer if the function were called `handleSelectDisplayOptionsTab` or `handleMakeDisplayOptionsTabActive` and the bool called `isDisplayOptionsTabActive`, as my initial impression of this function was that it would be saving my display options (which it doesn't)